ocs_ci/ocs/constants.py

```diff
@@ -629,3 +629,6 @@
 
 # gather bootstrap
 GATHER_BOOTSTRAP_PATTERN = 'openshift-install gather bootstrap --help'
+
+DEFAULT_AWS_REGION = 'us-east-2'
+CLUSTER_PREFIXES_TO_EXCLUDE_FROM_DELETION = ['jnk']
```

Comment:
Does this need to be in `constants.py`? This seems very specific to the aws cleanup code. I'm not sure it has to live in the constants module, as that was designed to hold values that are shared across multiple other modules.

Reply:
same as the above comment. Please let me know if any change is needed